frontend/src/container/ListAlertRules/AlertsEmptyState/alertLinks.ts

```diff
@@ -0,0 +1,50 @@
+export const ALERT_INFO_LINKS = [
+	{
+		infoText: 'How to create Metrics-based alerts',
+		link:
+			'https://signoz.io/docs/alerts-management/metrics-based-alerts/?utm_source=product&utm_medium=alert-empty-page',
+		leftIconVisible: false,
+		rightIconVisible: true,
+	},
+	{
+		infoText: 'How to create Log-based alerts',
+		link:
+			'https://signoz.io/docs/alerts-management/log-based-alerts/?utm_source=product&utm_medium=alert-empty-page',
+		leftIconVisible: false,
+		rightIconVisible: true,
+	},
+	{
+		infoText: 'How to create Trace-based alerts',
+		link:
+			'https://signoz.io/docs/alerts-management/trace-based-alerts/?utm_source=product&utm_medium=alert-empty-page',
+		leftIconVisible: false,
+		rightIconVisible: true,
+	},
+];
```

Comment:
@YounixM - I have not moved/created constant here for these strings has these wont be used in any case more than one time.

Reply:
It helps with code readability too.